src/signals/incident-management/containers/IncidentDetail/components/StatusForm/StatusForm.tsx

```diff
@@ -139,150 +172,180 @@ const StatusForm: FunctionComponent<StatusFormProps> = ({
     dispatch({ type: 'SET_TEXT', payload: event.target.value })
   }, [])
 
+  const defaultTextTemplatesLength = () => {
+    if (!defaultTexts || defaultTexts.length === 0) {
+      return 0
+    }
+    const statusDefaultTexts = defaultTexts.filter(
+      (text) => text.state === state.status.key
+    )
+    return statusDefaultTexts[0] ? statusDefaultTexts[0].templates?.length : 0
+  }
+
+  const useDefaultText = (event: SyntheticEvent, text: string) => {
```

Comment:
Same here; make this a `useCallback` call?

Reply:
✅